src/interfaces/decorator-metadata-bag.interface.ts

```diff
@@ -0,0 +1,27 @@
+import { TypeMetadata, ExposeMetadata, ExcludeMetadata, TransformMetadata } from '.';
+
+/**
+ * Information assigned to a class property or class (constructor) via decorators.
+ */
+export interface DecoratorMetaDataBag<T> {
+  /**
+   * Optional metadata assigned through the @Expose decorator.
+   */
+  expose: ExposeMetadata[];
+
+  /**
+   * Optional metadata assigned through the @Exclude decorator.
+   */
+  exclude: ExcludeMetadata[];
+  /**
+   * Optional metadata assigned through the @Transform decorator. This value is always
+   * undefined for bags belonging to classes. (The decorator is a `PropertyDecorator`.)
```

Comment:
What is a bag?

Reply:
You basically store stuff in a bag. Most of the time with inconsistent properties. We can change naming later.

Reply:
Ok